app/Http/Controllers/Devices/DeviceController.php

```diff
@@ -82,6 +84,10 @@ public function update($id, DeviceUpdateRequest $request) {
         return $this->response->accepted();
```

Comment:
een update wordt niet async afgehandeld return $this->response->noContent(); past beter